app/routes/groups_.new.tsx

```diff
@@ -27,29 +28,64 @@ export async function action({ request }: ActionFunctionArgs) {
     return { error: { message: `Form not submitted correctly.` } };
   }
 
-  let groupImageUrl = null;
+  const formObject = Object.fromEntries(form);
```

Comment:
Can we remove the if statement above? It seems redundant as we now perform some checks twice (once in the if-statement and once with zod).